Packages/com.unity.inputsystem/InputSystem/Editor/Debugger/InputDebuggerWindow.cs

```diff
@@ -333,7 +333,7 @@ void ISerializationCallbackReceiver.OnAfterDeserialize()
             s_Instance = this;
         }
 
-        class InputSystemTreeView : TreeView
+        private class InputSystemTreeView : TreeView
```

Comment:
I asked before on slack, but what code convention are we following? (Because Unity's C# conventions says to drop these)

Reply:
In the end, I guess part of the answer is "mine".

Reply:
Yeah, I thought so. The problem with that is, it makes it hard for contributors to follow these guidelines if they exist only in your head. And from what I've seen in PRs, you are rather eager in having everyone follow these guidelines.

As a non-you committer, I'd prefer it if we could either:
-adopt a written down guideline.
or
-agree that this is not important enough to discuss in PRs.